src/components/Navbar/Navbar.js

```diff
@@ -8,9 +8,41 @@ import { ReactComponent as AddItem } from "../../img/add_item.svg";
 import { ReactComponent as ChangeList } from "../../img/change_list.svg";
 import { ReactComponent as ShareList } from "../../img/share_list.svg";
 
-import { Link } from "react-router-dom";
+import { Link, useHistory } from "react-router-dom";
+import swal from "@sweetalert/with-react";
 
 export default function Navbar(props) {
+  const history = useHistory();
+
+  const shareTokenHandler = () => {
+    const token = localStorage.getItem("token");
+    swal({
+      title: token,
+      text: "Use this token to share this shopping list with friends",
+      dangerMode: true,
+    });
+  };
+
+  const logoutHandler = () => {
+    swal({
+      title: "Do you want to log out?",
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    }).then((willLogout) => {
```

Comment:
This is great! Just thinking from a UX perspective about the copy: maybe we should be extra explicit about "write down this three-word token phrase to share access with friends or reaccess this list yourself" and "Do you want to eject this list and access another?" or something. Really nice work on the code!

Reply:
Good idea - I've updated the code!